R/round.r

```diff
@@ -159,10 +161,10 @@ parse_unit_spec <- function(unitspec) {
     mult <- as.numeric(parts[[1]])
     unit <- parts[[2]]
   }
-  
+
```

Comment:
> Is this PR almost finished? Do the tests and docs need to be modified to reflect this 

Unit test fail. 

Github doesn't show your indentation right. It's completely messed up. Are you using TABs to indent your code? Please use simple spaces. R studio must have an option for that. 

Also please squash all the commits into one.  You can add one test for some corner case.

Thanks.